packages/test/test-end-to-end-tests/src/test/pongEvent.spec.ts

```diff
@@ -0,0 +1,103 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { IRequest } from "@fluidframework/core-interfaces";
+import { useFakeTimers, SinonFakeTimers } from "sinon";
+import { IFluidCodeDetails } from "@fluidframework/container-definitions";
+import { Container, ConnectionState, Loader } from "@fluidframework/container-loader";
+
+import {
+	LocalCodeLoader,
+	TestObjectProvider,
+	LoaderContainerTracker,
+	TestContainerRuntimeFactory,
+	ITestObjectProvider,
+	TestFluidObjectFactory,
+	waitForContainerConnection,
+} from "@fluidframework/test-utils";
+import {
+	getDataStoreFactory,
+	TestDataObjectType,
+	describeNoCompat,
+} from "@fluidframework/test-version-utils";
+import { IContainerRuntimeBase } from "@fluidframework/runtime-definitions";
+
+const id = "fluid-test://localhost/containerTest";
+const testRequest: IRequest = { url: id };
+const codeDetails: IFluidCodeDetails = { package: "test" };
+const timeoutMs = 500;
+
+// REVIEW: enable compat testing?
+describeNoCompat("Pong", (getTestObjectProvider) => {
```

Comment:
we do not care about compat testing, as the test only validates behavior of one layer (for most part). Plus, test will fail today (if old driver is used), though it could be enabled in the future.